eng/liveBuilds.targets

```diff
@@ -224,6 +223,23 @@
             ResolveLibrariesRefAssembliesFromLocalBuild;
             ResolveLibrariesRuntimeFilesFromLocalBuild" />
 
+  <!--
+  Native files that are always included in the singlefile bundle can be listed here.
+
+  Example:
+    <SingleFileHostIncludeFilename Include="somefile.dll" />
+  -->
+  <ItemGroup>
```

Comment:
I moved this out of packaging so that it would also get used for the `pretest` live runtimepack that we generate. Without this, live publishing for single-file ends up with a lot of extra binaries in the output.